packages/gatsby-theme-docs/gatsby-config.js

```diff
@@ -210,6 +213,54 @@ module.exports = (themeOptions = {}) => {
       'gatsby-plugin-remove-trailing-slashes',
       'gatsby-plugin-meta-redirect',
       'gatsby-plugin-netlify-cache',
+      {
+        resolve: 'gatsby-plugin-feed',
+        options: {
+          query: `
+          {
+            site {
+              siteMetadata {
+                title
+                description
+                siteUrl
+                site_url: siteUrl
+              }
+            }
+          }
+        `,
+          feeds: [
+            {
+              serialize: ({ query: { site, allReleaseNotePage } }) => {
+                return allReleaseNotePage.nodes.map((node) => {
+                  return {
+                    ...node,
+                    url: site.siteMetadata.siteUrl + node.slug,
+                    guid: site.siteMetadata.siteUrl + node.slug,
+                    custom_elements: [{ 'content:encoded': node.body }],
+                  };
+                });
+              },
+              query: `
+              {
+                allReleaseNotePage(
+                  sort: { order: DESC, fields: date },
+                ) {
+                    nodes {
+                      description
+                      body
```

Comment:
If we don't find a solution for the encoding (see above) this field does not need to be queried any more.